src/main/resources/org/researchspace/apps/help/data/templates/http%3A%2F%2Fhelp.researchspace.org%2Fresource%2FStorage.html

```diff
@@ -163,8 +164,8 @@ <h3 id="runtime-s3-example">Example: S3 as <code>runtime</code> Storage</h3>
 -Dconfig.storage.runtime.endpoint=https://s3.amazonaws.com/ \
 -Dconfig.storage.runtime.signingRegion=us-east-1 \
```

Comment:
I think it is just `region` not `signingRegion`, could you please change it in the docs

Reply:
Fix in https://github.com/researchspace/researchspace/pull/207/commits/5fa15d72fb75923c535899b1c3da3056efa03f90